src/Plugin.php

```diff
@@ -56,6 +56,11 @@ public static function enqueueAssets() {
     if (!is_product()) {
       return;
     }
+
+    if (has_term(Product::CATEGORY_EXCLUDED, 'product_cat')) {
```

Comment:
Please see my comment in the conversation. I think your previous code was totally fine 👍